guac-graphql.md

```diff
@@ -94,10 +94,18 @@ utility that can take a collection of files and ingest them into the GUAC graph.
 In your original window, run:
 
 ```bash
-bin/guacone collect files ../guac-data/docs/
+bin/guacone collect --add-vuln-on-ingest --add-eol-on-ingest --add-license-on-ingest files ../guac-data/docs/
```

Comment:
Does `-add-eol-on-ingent` work as intended?

Reply:
It appears to, yes.

Reply:
Actually, it definitely does. We don't include the EOL collector in the compose file yet, so the data I had in my test instance had to come from the ingest-time argument.